config/peerpods/peerpodssecret.yaml

```diff
@@ -10,4 +10,4 @@ stringData:
   #LIBVIRT_NET: "default
   #LIBVIRT_POOL: "default"
   #REDHAT_OFFLINE_TOKEN: "" #Required to download rhel base image : Download token from https://access.redhat.com/management/api
-  
\ No newline at end of file
+  #HKD_CRT: "" #Download the certificate from https://www.ibm.com/support/resourcelink/api/content/public/host-key-documents.html
```

Comment:
The certificate is multi line, i'm not sure it will work as key's value in the CM, i think that in such cases usually best practice is to encode the file in base64 format and set as value or upload the file as standalone secret.
Having said that, can't we have the certificate embedded in the script rather than asking the user to set it? i.e. is this certificate frequently change ?

Reply:
Hi @snir911, The HKD is user dependent and differs for each user so the user should provide it, and it is working with the current flow where we've provided the HKD as a value in config map and we've tested the same by deploying a worload.

```
[root@bastion-sl ~]# oc get pods -n default
NAME             READY   STATUS    RESTARTS   AGE
osc-caa-commit   1/1     Running   0          100s

```

Reply:
hi @snir911  HOSTKEY is for per z Server. This will vary for customer to customer. Hence this has to be provided as input. Hence we are adding as secret.